lib/ReadsUtils/ReadsUtilsImpl.py

```diff
@@ -125,7 +126,7 @@ def _proc_upload_reads_params(self, params):
             kbtype = 'KBaseFile.PairedEndLibrary'
             single_end = False
         if revid:
-            interleaved = 0
+            interleaved = 1
```

Comment:
This can just be merged into the original interleaved line, something like
interleaved = 1 if params.get('interleaved') or revid else 0